Tests/web3swiftTests/localTests/ST20AndSecurityTokenTests.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  ST20AndSecurityTokenTests.swift
+//
+//  Created by JeneaVranceanu on 07.12.2022.
+//
+
+import XCTest
+import BigInt
+import Core
+
+@testable import web3swift
+
+class ST20AndSecurityTokenTests: XCTestCase {
+
+    var web3: Web3!
+    var ethMock: Web3EthMock!
+    var st20token: ST20!
+    var securityToken: SecurityToken!
+
+    override func setUp() async throws {
+        web3 = await Web3.InfuraGoerliWeb3(accessToken: Constants.infuraToken)
+        ethMock = Web3EthMock(provider: web3.provider)
+        web3.ethInstance = ethMock
+        st20token = ST20.init(web3: web3, provider: web3.provider, address: .contractDeploymentAddress())
+        securityToken = SecurityToken.init(web3: web3, provider: web3.provider, address: .contractDeploymentAddress())
+    }
```

Comment:
Correct me if I'm wrong: Is the all mocking happening just here, or I missing something?